ONSdigital/ssdc-rm-support-tool
# Don't call APIs if not authorised to do so

# Motivation and Context
There are two parts to securing a single-page application: 1. Don't display parts of the UI which the user isn't authorised to see, and 2. Don't call APIs which the user isn't authorised to use. We were doing the first, but not the second.

# What has changed
Added checks to make sure user is authorised before calling all APIs.

# How to test?
Try accessing the UI with the reduced permissions and check that it doesn't crash anywhere, now that it's not going to the backend for a bunch of data.

# Links
Trello: https://trello.com/c/HhliKdR2
